apps/myaccount/src/components/multi-factor-authentication/multi-factor-authentication.tsx

```diff
@@ -36,6 +36,14 @@ interface MfaProps extends SBACInterface<FeatureConfigInterface>, TestableCompon
     onAlertFired: (alert: AlertInterface) => void;
 }
 
+/**
+ * Converts the isReadOnlyUserString to a boolean variable
+ * @return {boolean} True/False
+ */
+const convertToBoolean = (isReadOnlyUserString: any): boolean => {
```

Comment:
Can we specify a type here?
```suggestion
const convertToBoolean = (isReadOnlyUserString: string): boolean => {
```

Reply:
Fixed by 7e96bf111989b4d81dd8a573bc5ceb710e62d15d.